packages/reports/addon/components/filter-values/dimension-select.ts

```diff
@@ -110,16 +123,32 @@ export default class DimensionSelectComponent extends Component<DimensionSelectC
    */
   @task({ restartable: true })
   *searchDimensionValues(term: string): TaskGenerator<NaviDimensionModel[] | undefined> {
+    const { dimensionColumn } = this;
     const searchTerm = term.trim();
     this.searchTerm = searchTerm;
     if (!searchTerm) {
       return undefined;
     }
+    const dimension: NaviDimensionModel[] = [];
+    const dimensionModelFactory = getOwner(this).factoryFor('model:navi-dimension');
+    const value = (('"' + searchTerm.toLowerCase()) as string | number) + '"';
+    dimension.push(
+      dimensionModelFactory.create({
+        value,
```

Comment:
Can we make `value` the value without the quotes and add the quotes in the template when `manualInputEntry === true`?